fdbserver/storageserver.actor.cpp

```diff
@@ -5875,6 +5875,23 @@ ACTOR Future<std::unordered_map<Key, Version>> dispatchChangeFeeds(StorageServer
 	}
 }
 
+namespace {
+ACTOR Future<Version> getRawReadVersion(Transaction* pTransaction) {
+	loop {
```

Comment:
@liquid-helium what if this call loops indefinitely when GRV keeps returning error. Will this cause any issue?

Reply:
We should avoid that.